packages/web-components/src/components/ic-text-field/ic-text-field.stories.mdx

```diff
@@ -918,3 +918,21 @@ import readme from "./readme.md";
     }
   </Story>
 </Canvas>
+
+### Logging icChange
+
+<Canvas>
+  <Story name="Logging icChange">
```

Comment:
This story can probably be skipped as the screenshot will just display an empty text field which we have already covered.